### PR TITLE
Add the ability to use non-default memcached server in the tests

### PR DESCRIPTION
**Pull request by alexkart**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Tests pass?   | ✔️
| Fixed issues  |

### CODE REVIEW

**Comment by alexkart** [comment]
Something is broken in the tests, not related to my changes.

File: tests/MemcachedTest.php [hunk 1]
```diff
@@ -1,32 +1,44 @@
...
+ * If memcached server is not running on localhost at port 11211 set environment variables MEMCACHED_HOST and
+ * MEMCACHED_PORT to use different host and port of the memcached server in the tests
  * @group memcached
  * @group caching
  */
 class MemcachedTest extends CacheTest
 {
+    public static $MEMCACHED_HOST;
+
+    public static $MEMCACHED_PORT;
```
Comment (rob006) [inline tests/MemcachedTest.php]: Why these properties does not follow naming conventions for properties?

Comment (samdark) [reply tests/MemcachedTest.php]: Yes, these should be memcachedHost and memcachedPort. Also I don't see why these should be public.

**Comment by samdark** [comment]
Unmergeable with current code.

**Pull request closed without merging**